src/ReachSets/discretize.jl

```diff
@@ -1,310 +1,661 @@
+const LDS = LinearDiscreteSystem
+const CLCDS = ConstrainedLinearControlDiscreteSystem
+
+@inline Id(n) = Matrix(1.0I, n, n)
+
 """
-    discretize(cont_sys, δ; [approx_model], [pade_expm], [lazy_expm], [lazy_sih])
+    discretize(𝑆, δ; [approximation], [exp_method], [sih_method])
+
+Apply an approximation model to `S` obtaining a discrete initial value problem.
+
+### Input
+
+- `𝑆`             -- initial value problem for a continuous affine ODE with
+                     non-deterministic inputs
+- `δ`             -- step size
+- `approximation` -- the method to compute the approximation model for the
+                     discretization, choose among:
+
+    - `"forward"`    -- use forward-time interpolation
+    - `"backward"`   -- use backward-time interpolation
+    - `"firstorder"` -- use first-order approximation of the ODE
+    - `"nobloating"` -- do not bloat the initial states
+
+- `exp_method`  -- (optional, default: `"base"`) the method used to take the matrix
+                    exponential of the coefficient matrix, choose among:
+
+    - `"base"` -- the scaling and squaring method implemented in Julia base,
+                  see `?exp` for details
+    - `"pade"` -- use Pade approximant method to compute matrix exponentials of
+                  sparse matrices, implemented in `Expokit`
+    - `"lazy"` -- compute a wrapper type around the matrix exponential, i.e. using
+                  the lazy implementation `SparseMatrixExp` from `LazySets` and
+                  the evaluation of the action of the matrix exponential using the
+                  `expmv` implementation from `Expokit`
+
+- `sih_method`  -- (optional, default: `"lazy"`) the method used to take the
+                    symmetric interval hull operation, choose among:
```

Comment:
This is inconsistent. The previous default and the new default in the options is `"concrete"`. We also know that `"lazy"` does not have any positive effect, so we should not use it by default.

Reply:
Good catch. Yes, i'll write `"concrete"`.